### PR TITLE
Ch giffy lube html & Nav bar event listeners

### PR DESCRIPTION
**Pull request by chesney-hardin**
Made some changes to the transientState module
-added a spread operator to make a copy of our initial transient state
-added a function to reset the transientState back to the initial state
-added a getter function for the whole transientState object

NavBar Event Listeners
-4 click event listeners added
-the state changed should show up in the console log (especially take note of when you logout and transientState is reset)
-the page does not actually change yet, because we need to hook it up in GiffyLube to know which view we want to change to

Giffy Lube HTML:
-conditionals for the page view are set up for view messages, message form, and default view. (still need one for post entry)

### CODE REVIEW

File: scripts/GiffyLube.js [hunk 1]
```diff
@@ -8,29 +12,49 @@ import { NavBarHTML } from "./nav/NavBar.js"
...
-    htmlList += `
-    <h1>This will be the main page</h1>
-    `
+
+    //check these conditions of the transientState to determine which page to generate 
+        //1. If transientState.view===viewMessages, then invoke function that generates all the HTML for the inbox page and add it to the htmlString
+        if(view === "viewMessages") {
+            htmlList += `<section class="messages
+                    >${messages}<
+                    /section>`
```
Comment (ryanmphill) [inline scripts/GiffyLube.js:32]: The browser is reading the </section> HTML as < /section>. If we get rid of the extra space everything else looks good!

File: scripts/GiffyLube.js [hunk 2]
```diff
@@ -8,29 +12,49 @@ import { NavBarHTML } from "./nav/NavBar.js"
...
+    //Create variable for htmlList = NavBar 
     let htmlList = `${navBar}`
-    htmlList += `
-    <h1>This will be the main page</h1>
-    `
+
+    //check these conditions of the transientState to determine which page to generate 
+        //1. If transientState.view===viewMessages, then invoke function that generates all the HTML for the inbox page and add it to the htmlString
+        if(view === "viewMessages") {
+            htmlList += `<section class="messages
```
Comment (ryanmphill) [inline scripts/GiffyLube.js:30]: "messages" class needs a closing quote

**Review by ryanmphill — CHANGES_REQUESTED** [review]

File: scripts/GiffyLube.js (hunk 1 above)
Comment (chesney-hardin) [reply scripts/GiffyLube.js:32]: changes made!

File: scripts/GiffyLube.js (hunk 2 above)
Comment (chesney-hardin) [reply scripts/GiffyLube.js:30]: changes made!

**Pull request merged**